hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -24,6 +24,8 @@ parameters:
   - name: FEATURE_FLAGS
     value: ""
     required: true
+  - name: ENABLE_UPGRADEPOLICY_STATUS
+    required: true
```

Comment:
add a default value "true" too. We can override it from the saas file in app interface